Classes/Service/ExportService.php

```diff
@@ -302,28 +344,27 @@ function (NodeData $node1, NodeData $node2) {
      *
      * @param array<NodeData> $nodeDataList The nodes to export
      * @return void The result is written directly into $this->xmlWriter
+     * @throws \Neos\ContentRepository\Exception\NodeTypeNotFoundException
      */
     protected function exportNodeDataList(array &$nodeDataList)
     {
-        $this->xmlWriter->startElement('nodes');
         $this->xmlWriter->writeAttribute('formatVersion', self::SUPPORTED_FORMAT_VERSION);
 
         $currentNodeDataIdentifier = null;
         foreach ($nodeDataList as $nodeData) {
             $this->writeNode($nodeData, $currentNodeDataIdentifier);
         }
-
-        $this->xmlWriter->endElement();
     }
 
     /**
      * Write a single node into the XML structure
      *
      * @param NodeData $nodeData The node data
-     * @param string $currentNodeDataIdentifier The "current" node, as passed by exportNodeDataList()
+     * @param string|null $currentNodeDataIdentifier The "current" node, as passed by exportNodeDataList()
      * @return void The result is written directly into $this->xmlWriter
+     * @throws \Neos\ContentRepository\Exception\NodeTypeNotFoundException
      */
-    protected function writeNode(NodeData $nodeData, &$currentNodeDataIdentifier)
+    protected function writeNode(NodeData $nodeData, string &$currentNodeDataIdentifier = null)
```

Comment:
I guess this is more fitting:
```suggestion
    protected function writeNode(NodeData $nodeData, ?string &$currentNodeDataIdentifier)
```